### ISSUE TITLE
When `allowMultiple` is disabled, display error when dropping multiple files

### ISSUE THREAD

**Issue by enisdenjo** [pqina/filepond#137]
This is a **feature** request.

Filepond with `allowMultiple=false` displays no error at all if multiple items are drag-and-dropped. They just disappear without any feedback.

The user will end up blaming the app, or writing support tickets; instead of noticing that they can drag-and-drop only one file.

**Comment by rikschennink** [comment]
Good point, I've been meaning to introduce some generic visual error state but haven't gotten around to it. A good start would be to at least communicate the error state via callback so a custom error message can be shown.

**Comment by minuz** [comment]
Adding to this request, the **maxFiles** seems to have a similar effect.
If I set `allowMultiple=true` and `maxFiles=5`, then when dragging more than 5, there's no feedback on the screen and it looks like a bug. Clicking from the file browser has a similar effect.

**Comment by mrksoftware** [comment]
Hi! Any news on this?

**Comment by rikschennink** [comment]
Still on the list, bugs have priority, still want to add this but haven’t had time yet

**Comment by SaschaHeyer** [comment]
Hi @rikschennink 
I agree with @minuz. When uploading more files then maxFiles nothing happens, a message for the user would be helpful. Do we have any other way to handle this?

Thank you for your work on this product =)

**Comment by rikschennink** [comment]
@SaschaHeyer Thanks! 

Until this is default functionality you can use `onwarning` to capture the error and render a warning yourself.

**Comment by SaschaHeyer** [comment]
@rikschennink great that's working

**Comment by fooramu** [comment]
Thank you for this issue and your solution approach.
